www/src/pages/blog/astro-018.astro

```diff
@@ -0,0 +1,112 @@
+
+          Named slots are particularly useful for layout components. Layout components are what we call components that provide common HTML for a page, such as commonly added CSS in the &lt;head&gt; and the page's skeleton content.
+
+          Now, a Layout can specify slots for content to go into the "head" section, and some to go into the body's "content" section. The layout component defines its slots as part of its API. It can have as many or as few as you need!
+
+          Named slots will allow for **more robust** and dynamic Astro components. This will especially be useful for distributing components as packages where components take more options and are less opinionated.
+
+          ![Solid.js logo](/assets/blog/astro-018/solid-logo.png)
+
+          ## Solid support
+
+          [Solid](https://www.solidjs.com/) is a JSX-based reactive framework that compiles to efficient DOM manipulation. Before Astro was even public, the Solid team expressed interest in building a renderer. With the new [Solid renderer](https://github.com/snowpackjs/astro/tree/main/packages/renderers/renderer-solid), now you can use Solid as a first-class framework in your project.
+
+          Even though Solid implements its own JSX compilation that differs from React and Preact, you're still able to use it side-by-side with any of those framework components. This is the promise of Astro, use all of your favorite framework(s)!
+
+          ![Lit logo](/assets/blog/astro-018/lit-logo.svg)
+
+          ## Lit support
+
+          The ability to server render web components is finally arriving thanks to [Declarative Shadow DOM](https://web.dev/declarative-shadow-dom/), a new HTML feature that's now available in Chrome. [Lit](https://lit.dev/) has experimental support for server rendering and we've brought that into Astro via the new [Lit renderer](https://github.com/snowpackjs/astro/tree/main/packages/renderers/renderer-lit).
+
+          This allows you to define a Lit element in a JavaScript file and then render it in Astro, just as you would a framework component like React or Vue. One major difference is that you use the custom element tag name, just as you would in HTML. When doing that you get:
+
+          * Fully rendered custom elements!
+          * Automatic inclusion of the declarative shadow DOM polyfill (for non-Chromium browsers).
+          * Integration with Lit's hydration system.
+
+          The Lit renderer is marked as experimental, but we're already excited to see that many popular libraries render correctly with it.  As other web component libraries adopt declarative shadow DOM and common APIs emerge for rendering, we hope to bring support to those libraries as well.
+
+
+          ![Example showing using the style global tag to provide global CSS classes for the entire page.](/assets/blog/astro-018/style-global.png)
+
+          ## &lt;style global&gt;
```

Comment:
this section and the one below it feel a bit light on content. Maybe combine into a single "Try out Astro v0.18" section that includes a bulleted list of "other cool stuff" and include `style global`, new docs site, and other highlights.

Reply:
Or, if this is mentioned in the list of features at the top of the site (see previous comment) you may not even need this section at all.

Reply:
Noticed you removed this one (and the docs one). I don't think short sections are bad. The Next.js blog post you referenced had a lot of short sections.

Reply:
Maybe remove the banner image then? It felt weird to me as a reader to have the title + section image take up more space than the content of the section :)